src/components/Proposals/InfoBoxes.js

```diff
@@ -14,7 +14,7 @@ import { dateFormat, toMs } from '../../lib/date-utils'
 import { round, safeDiv } from '../../lib/math-utils'
 import SummaryBar from './SummaryBar'
 
-function InfoBoxes({ vote }) {
+function InfoBoxes({ vote, disabled }) {
```

Comment:
I don't think this is a problem yet, but if we see ourselves drilling lots of props down to these components it might be worth flattening it out a providing each component with their props directly from within `ProposalDetail`